index.html

```diff
@@ -8,15 +8,19 @@
 </head>
 
 <body>
-  <div id="main">
-    <h1 id="title">Dr. Norman Borlaug</h1>
+  <main id="main">
+    <header>
+      <h1 id="title">Dr. Norman Borlaug</h1>
+    </header>
     <p id ="title-description">The man who saved a billion lives</p>
-    <div id="img-div">
-      <img src="image/tribute-page-main-image (1).jpg" alt=" tribut image" id="image">
-      <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
-        yields - part of his life-long war on hunger.</p>
-      </div>
-      <div id="tribute-info">
+    <section id="img-div">
+      <figure>
+        <img src="image/tribute-page-main-image (1).jpg" alt=" tribut image" id="image">
+        <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat
+          yields - part of his life-long war on hunger.</p>
+      </figure>
```

Comment:
Please replace paragraph  element with figcaption element. it will be difficult for web visitor with screen reader to tell the difference between paragraph  and alt text. it is advisable to use  figcaption element along with figure element to add caption to image.